src/local-editor-service.ts

```diff
@@ -98,21 +97,45 @@ export class LocalEditorService extends EditorService {
           };
         })
       };
-    } else if (location.kind === 'attribute') {
+    }
+
+    if (location.kind === 'attributeValue') {
+      const domModules = document.getByKind('dom-module');
+      if (domModules.size > 1) return;
```

Comment:
This line prohibits parsing of a file with 2 `dom-module`s. Reason being that `parse5` does not provide a `parentNode` for a `template`. Therefore traversing the ancestors inside a `template` of a `dom-module` will halt at the `template`. This is also the case for the function `nodeWalkAncestors` in `dom5` which just checks the `parentNode`